libs/wingsdk/src/std/string.ts

```diff
@@ -41,6 +41,8 @@ export class String {
   /**
    * Returns the character at the specified index.
    *
+   * @macro ((args) => { if ($args$ >= $self$.length) {throw new Error("index out of bounds")}; return $self$.at($args$) })($args$)
```

Comment:
what about `"boom".at(-5)`?  (in js is undefined) 
Should it throw the same error?

Reply:
yes

Reply:
done 👌✅